### ISSUE TITLE
`.title` class breaks `*-justify` attributes

### ISSUE THREAD

**Issue by zerodevx** [PolymerElements/paper-toolbar#37]
Not sure if it's intended behaviour - adding `.title` classes breaks `paper-toolbar`'s justification attributes.

Say I have

``` html
<paper-toolbar justify="center">
  <div class="title">x-test</div>
</paper-toolbar>
```

We'd expect "x-test" to be in the center but it's left-aligned. Remove `.title` class and it works.

Repro: http://jsbin.com/qibelulexu/edit?html,output

**Comment by blasten** [comment]
That is correct. title will override the behavior of the justify attribute. For now, you may need to create your own custom-title.
